test/mlj_interface.jl

```diff
@@ -52,10 +52,6 @@
             for data in [MLJTestInterface.make_binary(), MLJTestInterface.make_multiclass()]
```

Comment:
This is kind of orthogonal, but FYI, to test row-based tables as well as column-based ones, we can now do this (with MLJTestInterface.jl 0.2.4):

```suggestion
            for data in [
                MLJTestInterface.make_binary(), 
                MLJTestInterface.make_multiclass(),
                MLJTestInterface.make_binary(row_table=true), 
                MLJTestInterface.make_multiclass(row_table=false),
            ]
```